### PR TITLE
email confirmation added

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
Email confirmation for admins

Fixes #844 

# Type of Change:
- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by anjali-dhanuka** [comment]
Hi @Monal5031 @abhishekarya286 @kamsuri @nida Please review

File: vms/pom/pages/adminRegistrationPage.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class AdminRegistrationPage(BasePage):
     INVALID_PHONE = 'Please enter a valid phone number'
     INVALID_PHONE_FOR_COUNTRY = 'This phone number isn\'t valid for the selected country'
     NO_MATCH = 'Passwords don\'t match.'
+    confirm_email_message = "Please confirm your email address before login."
```
Comment (Monal5031) [inline vms/pom/pages/adminRegistrationPage.py]: all caps, keep it consistent.

**Comment by abhishekarya286** [comment]
Update the PR description.

**Comment by anjali-dhanuka** [comment]
@Monal5031 I am done with the change you suggested. Please  review

**Pull request merged**